docs/introduction/flows.md

```diff
@@ -30,36 +30,36 @@ Intents, Commitments, and Economic Events can occur at any granularity that is n
 
 All flows use an action property to designate what the flow is doing and how it will affect an economic resource (or not).  Actions are defined as follows.
 
+* vf:produce - new resource created in that process or an existing stock resource added to
 * vf:use - for example a tool used in process, after the process, the tool still exists
 * vf:consume - for example an ingredient composed into the output, after the process the ingredient is gone
 * vf:cite - for example a design file, neither used nor consumed, the file remains available at all times
 * vf:work - labor power towards a process
-* vf:accept - in processes like repair or maintentance, same resource will appear in output with *vf:improve* verb
-* vf:load -  transported resource enters the process, same resource will appear in output with *vf:unload* verb
-* vf:produce - new resource created in that process
-* vf:improve - in processes like repair or maintentance, same resource will appear in input with *vf:accept* verb
-* vf:unload -  transported resource leaves the process, same resource will appear in input with *vf:load* verb
-
-Action | Affect | Related to | I/O | Pairs with |
------- | ------ | ---------- | --- | ---------- |
-increment | Increment | N/A | N/A | N/A |
-produce | Increment | Process | Output | N/A |
-improve | No effect | Process | Output | accept |
-unload | Increment | Process | Output | load |
-receive | Increment | Transfer | Output | give |
-decrement | Decrement | N/A | N/A | N/A |
-load | Decrement | Process | Input | unload |
-consume | Decrement | Process | Input | N/A |
-use | No effect | Process | Input | N/A |
-work | No effect | Process | Input | N/A |
-cite | No effect | Process | Input | N/A |
-accept | No effect | Process | Input | improve |
-give | Decrement | Transfer | Input | receive |
+* vf:load -  transported resource enters the process, the same resource will appear in output with *vf:unload* verb
+* vf:unload -  transported resource leaves the process, the same resource will appear in input with *vf:load* verb
+* vf:accept - in processes like repair or maintentance, the same resource will appear in output with *vf:improve* verb
+* vf:improve - in processes like repair or maintentance, the same resource will appear in input with *vf:accept* verb
+* vf:give - give rights and responsibilities for the resource
+* vf:receive - receive rights and responsibilites for the resource
+* vf:raise - adjusts a quantity up based on beginning balance or inventory account adjustment
+* vf:lower - adjusts a quantity down based on beginning balance or inventory account adjustment
+
+Action | Affect | Related to | I/O | Changes existence | Pairs with |
+------ | ------ | ---------- | --- | ----------------- | ---------- |
```

Comment:
latest version misses *increment* and *decrement*

Reply:
actually, i'd like to do a PR and make this "reified" - if I am using the term correctly - add other properties to vf:Action, which would be a class instead of a property - along the lines of this chart - what do you think?

Reply:
I think this would result in adding bunch of terms to the vocab just for the sake of represent that table, while this table seems mostly affecting implementations and not processing of data at run-time. I would prefer to finish this PR and if you see it a priority you could start another one but preferably not as part of *cleanup* milestone but as place to give consideration to this possibility.

Reply:
Agree, I didn't want to do that PR for this cleanup release.  Just explaining part of why I didn't want to include increment and decrement as actions in their own right.